app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(persons: list) -> list:
+    person_array = []
+    for person in persons:
+        person_array.append(Person(name=person["name"], age=person["age"]))
+    for person in persons:
+        human = Person.people[person["name"]]
+        if person.get("wife") is not None:
+            human.wife = Person.people[person["wife"]]
+        elif person.get("husband") is not None:
```

Comment:
Maybe people have both?) wife and husband change elif to if

Reply:
did not understand the comments, please repeat more understandable to me